packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -85,10 +85,11 @@ export const MostReadLink = ({
   script,
   title,
   href,
+  small,
   children,
 }) => (
   <StyledItem dir={dir}>
-    <StyledLink href={href} script={script} service={service}>
+    <StyledLink href={href} script={script} service={service} small={small}>
```

Comment:
How about this instead: `typography="GelPrimer"` e.g.
```suggestion
    <StyledLink href={href} script={script} service={service} typography={typography}>
```

Reply:
Thanks @sareh. For the GEL typography Type sizes, is it the case that Group C is not used in the codebase?